core/templates/dev/head/services/utilsService.js

```diff
@@ -17,23 +17,21 @@
  * used at multiple places in the codebase.
  */
 
-oppia.factory('utilsService', ['UrlInterpolationService',
-  function(UrlInterpolationService) {
-    var utils = {
-      isEmpty: function(obj) {
-        for (var property in obj) {
-          if (obj.hasOwnProperty(property)) {
-            return false;
-          }
+oppia.factory('utilsService', [function() {
+  var utils = {
+    isEmpty: function(obj) {
+      for (var property in obj) {
```

Comment:
the currently implemented method of checking for an empty object seems to be fine. Also, there is a caveat in checking for length of object.keys:
```js
console.log(Object.keys(new Date()).length === 0); // true
```